packages/trace-viewer/src/snapshotRenderer.ts

```diff
@@ -223,7 +223,16 @@ function snapshotScript(...targetIds: (string | undefined)[]) {
         scrollLefts.push(e);
 
       for (const element of root.querySelectorAll(`[__playwright_value_]`)) {
-        (element as HTMLInputElement | HTMLTextAreaElement).value = element.getAttribute('__playwright_value_')!;
+        const value = element.getAttribute('__playwright_value_')!;
+        const inputElement = element as HTMLInputElement;
+        if (inputElement.type === 'file') {
+          const file = new File([new Blob([''], { type: 'text/plain' })], value.split(/(\/|\\)/).pop()!, { type: 'text/plain' });
```

Comment:
Let's just ignore it. I don't think anyone needs a file there. It also probably does not work in WebKit.